client/js/map.js

```diff
@@ -232,13 +235,22 @@ function showMarkerInfo(point, marker, index) {
     $("#tree_next").off('click').on('click', function () {
         fetchMarkers = false;
         var index = parseInt($(this).val(), 10)
+        panelLoader.classList.add('active');
         showMarkerInfoByIndex(index)
+        window.setTimeout(function () {
+          panelLoader.classList.remove('active');
+        }, 2000)
```

Comment:
Can we do something here other than setting a 2 second timeout?  The images sometime load faster than 2 seconds (and will be getting faster with once they are on our CDN), we want the loading icon to be removed as soon as the image is loaded.

Reply:
@deepwinter I went ahead and removed the timeout. The loading indicator now is triggered by the `load` event on the image in commit ffb85ff